.github/workflows/main.yml

```diff
@@ -0,0 +1,65 @@
+name: Main
```

Comment:
can we add to the workflow (or have another one?) for rubocop? 

example: https://github.com/rails/rails/actions?query=workflow%3ARuboCop

Reply:
Of course, we might have to create a list of functions.